src/models/User.ts

```diff
@@ -7,7 +7,10 @@ export interface IUser {
   projects: string[];
 }
 
-export interface UserDocument extends IUser, Document {}
+export interface UserDocument extends IUser, Document {
+  addProject(projectId: string): void;
+  deleteProject(projectId: string): void;
+}
 export interface UserModel extends Model<UserDocument> {
```

Comment:
혹시 model 건들이신 김에 컨벤션 변경된거 적용해주실수 있나요???
```suggestion
export interface IUserDocument extends IUser, Document {
  addProject(projectId: string): void;
  deleteProject(projectId: string): void;
}
export interface UserModel extends Model<IUserDocument> {
```